### PR TITLE
[chore] Delete pre-migration code

### PR DESCRIPTION
**Pull request by sanjayprabhu**
## Motivation

Deleted unused code around signers and old L1 Id/Name registry events

## Change Summary

Describe the changes being made in 1-2 concise sentences.

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [x] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#35-adding-changesets)
- [x] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [ ] PR includes [documentation](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#32-writing-docs) if necessary.
- [x] All [commits have been signed](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#22-signing-commits)

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

---

## PR-Codex overview
This PR focuses on deleting deprecated RPC calls and events. 

### Detailed summary
- Deleted packages/core/src/signers/types.ts
- Deleted protobufs/schemas/id_registry_event.proto
- Deleted protobufs/schemas/name_registry_event.proto
- Deleted apps/hubble/src/storage/db/idRegistryEvent.ts
- Deleted apps/hubble/src/storage/stores/signerStore.ts
- Deleted apps/hubble/src/storage/db/idRegistryEvent.test.ts
- Deleted apps/hubble/src/storage/stores/signerStore.test.ts
- Deleted apps/hubble/src/storage/db/nameRegistryEvent.test.ts
- Deleted packages/core/src/signers/index.ts
- Deleted IdRegistryEventRequest message from protobufs/schemas/request_response.proto
- Deleted `if (type === MessageType.SIGNER_ADD || type === MessageType.SIGNER_REMOVE)` block from apps/hubble/src/storage/db/message.ts
- Replaced `Factories.SignerAddMessage` with `Factories.CastAddMessage` in apps/hubble/src/network/utils/factories.ts
- Deleted `submitIdRegistryEvent` and `submitNameRegistryEvent` methods from apps/hubble/src/test/mocks.ts
- Deleted `id_registry_event` import from protobufs/schemas/gossip.proto
- Deleted `pruneCount2` expectation in apps/hubble/src/storage/stores/store.test.ts
- Replaced `RootPrefix.IdRegistryEvent` with `Buffer.from([RootPrefix.OnChainEvent, OnChainEventPostfix.IdRegisterByFid])` in apps/hubble/src/network/sync/syncEngine.ts
- Deleted `idRegistryEventToLog` and `nameRegistryEventToLog` functions from apps/hubble/src/utils/logger.ts
- Replaced `FARCASTER_EPOCH` import with `putOnChainEventTransaction` in apps/hubble/src/storage/stores/reactionStore.test.ts, apps/hubble/src/storage/stores/userDataStore.test.ts, apps/hubble/src/storage/stores/linkStore.test.ts, apps/hubble/src/storage/stores/castStore.test.ts, apps/hubble/src/storage/stores/usernameProofStore.test.ts
- Replaced `RootPrefix.IdRegistryEventByCustodyAddress` and `RootPrefix.NameRegistryEventsByExpiry` with `RootPrefix.OnChainEvent` in apps/hubble/src/profile/profile.ts
- Deleted `SignerAddMessageFactory` and `SignerRemoveMessageFactory` tests from packages/core/src/factories.test.ts
- Deleted `signMessageHash` test case from packages/core/src/crypto/ed25519.test.ts

> The following files were skipped due to too many changes: `packages/core/src/crypto/ed25519.test.ts`, `apps/hubble/src/storage/stores/utils.ts`, `apps/hubble/src/rpc/adminServer.ts`, `apps/hubble/src/console/genCommand.ts`, `apps/hubble/src/storage/engine/seed.ts`, `apps/hubble/src/profile/rpcProfile.ts`, `apps/hubble/src/storage/db/types.ts`, `protobufs/schemas/hub_event.proto`, `protobufs/schemas/message.proto`, `apps/hubble/src/rpc/test/submitService.test.ts`, `packages/hub-nodejs/src/generated/request_response.ts`, `packages/core/src/protobufs/generated/request_response.ts`, `packages/core/src/protobufs/types.ts`, `packages/core/src/builders.ts`, `apps/hubble/src/network/sync/syncEnginePerf.test.ts`, `apps/hubble/src/rpc/test/userDataService.test.ts`, `apps/hubble/src/storage/stores/storageCache.test.ts`, `packages/core/src/protobufs/typeguards.ts`, `apps/hubble/src/rpc/test/verificationService.test.ts`, `protobufs/schemas/rpc.proto`, `apps/hubble/src/storage/stores/userDataStore.ts`, `apps/hubble/src/rpc/test/syncService.test.ts`, `apps/hubble/src/storage/stores/storeEventHandler.test.ts`, `apps/hubble/src/rpc/test/linkService.test.ts`, `apps/hubble/src/utils/periodicTestDataJob.ts`, `apps/hubble/src/hubble.ts`, `apps/hubble/src/rpc/test/httpServer.test.ts`, `packages/core/src/validations.ts`, `apps/hubble/src/rpc/test/reactionService.test.ts`, `apps/hubble/src/storage/stores/storeEventHandler.ts`, `packages/core/src/protobufs/generated/gossip.ts`, `apps/hubble/src/network/p2p/gossipNode.test.ts`, `apps/hubble/src/rpc/test/castService.test.ts`, `packages/core/src/validations.test.ts`, `apps/hubble/src/storage/db/nameRegistryEvent.ts`, `packages/core/src/builders.test.ts`, `apps/hubble/src/console/warpcastTestCommand.ts`, `packages/hub-web/src/generated/request_response.ts`, `apps/hubble/src/rpc/test/rpcAuth.test.ts`, `apps/hubble/src/rpc/test/server.test.ts`, `apps/hubble/src/rpc/test/bulkService.test.ts`, `packages/core/src/factories.ts`, `apps/hubble/src/rpc/server.ts`, `apps/hubble/src/rpc/test/eventService.test.ts`, `packages/hub-web/src/generated/message.ts`, `packages/hub-nodejs/src/generated/message.ts`, `packages/core/src/protobufs/generated/message.ts`, `packages/hub-web/src/generated/hub_event.ts`, `packages/hub-nodejs/src/generated/hub_event.ts`, `packages/core/src/protobufs/generated/hub_event.ts`, `apps/hubble/src/network/sync/syncEngine.test.ts`, `apps/hubble/src/rpc/test/signerService.test.ts`, `apps/hubble/src/storage/engine/index.ts`, `packages/hub-nodejs/src/generated/rpc.ts`, `apps/hubble/src/network/sync/multiPeerSyncEngine.test.ts`, `packages/hub-web/src/generated/rpc.ts`, `apps/hubble/src/storage/engine/index.test.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/core/src/protobufs/generated/message.ts [hunk 1]
```diff
@@ -91,11 +91,11 @@ export enum MessageType {
   VERIFICATION_ADD_ETH_ADDRESS = 7,
   /** VERIFICATION_REMOVE - Remove a Verification */
   VERIFICATION_REMOVE = 8,
-  /** SIGNER_ADD - Add a new Ed25519 key pair that signs messages for a user */
-  SIGNER_ADD = 9,
-  /** SIGNER_REMOVE - Remove an Ed25519 key pair that signs messages for a user */
-  SIGNER_REMOVE = 10,
-  /** USER_DATA_ADD - Add metadata about a user */
+  /**
+   * USER_DATA_ADD - Deprecated
```
Comment (sds) [inline packages/core/src/protobufs/generated/message.ts:95]: I know this was auto-generated, but this comment looks wrong?

File: packages/core/src/validations.ts [hunk 2]
```diff
@@ -156,7 +153,7 @@ export const validateMessage = async (
     return err(new HubError("bad_request.validation_failure", "signer is missing"));
   }
 
-  const eip712SignerRequired = EIP712_MESSAGE_TYPES.includes(data.type);
+  const eip712SignerRequired = false;
```
Comment (sds) [inline packages/core/src/validations.ts:156]: I understand we might want to re-introduce a message type that requires this kind of signature in future, but given we don't have anything planned, might be best to just remove? Not a big deal.

**Review by sds — COMMENTED** [review]
Overall looks fine, just two minor comments.

File: packages/core/src/protobufs/generated/message.ts (hunk 1 above)
Comment (sanjayprabhu) [reply packages/core/src/protobufs/generated/message.ts:95]: Huh, not sure why it did this. It is autogenerated, I don't want to fix this manually. Will investigate more when I have some time, but will just merge as is for now.

**Pull request merged**